project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.html

```diff
@@ -239,6 +239,53 @@
                 [isLoading]="insightsDataLoading"></ws-widget-weekly-claps>
             </div>
 
+            <div class="assessments-wrapper" *ngIf="assessmentsData">
+              <div class="assessments-info-wrapper">
+                  <div class="flex justify-between">
+                      <div class="flex flex-col">
+                          <div class="flex flex-between flex-middle">
+                              <p class="flex-align-start upcoming-assessment-title flex flex-middle margin-remove">Upcoming Assessment
```

Comment:
please apply transalation for the same

Reply:
We commented this, while enabling i can do the transaltion.